commands/scan/scan.go

```diff
@@ -469,6 +522,28 @@ func appendErrorSlice(scanErrors []formats.SimpleJsonError, errorsToAdd [][]form
 	return scanErrors
 }
 
+func depsListFromVulnerabilities(scanResult ...services.ScanResponse) []string {
+	var depsList []string
+	var technologiesList []coreutils.Technology
+	for _, result := range scanResult {
+		for _, vulnerability := range result.Vulnerabilities {
+			dependencies := maps.Keys(vulnerability.Components)
+			for _, dependency := range dependencies {
+				if !slices.Contains(depsList, dependency) {
+					depsList = append(depsList, dependency)
+				}
+			}
+
+			if !slices.Contains(technologiesList, coreutils.Technology(vulnerability.Technology)) && (vulnerability.Technology != "") {
+				technologiesList = append(technologiesList, coreutils.Technology(vulnerability.Technology))
+			}
+
+		}
+
+	}
+	return depsList
+}
```

Comment:
```suggestion
func depsListFromVulnerabilities(scanResult ...services.ScanResponse) (depsList []string) {
	for _, result := range scanResult {
		for _, vulnerability := range result.Vulnerabilities {
			dependencies := maps.Keys(vulnerability.Components)
			for _, dependency := range dependencies {
				if !slices.Contains(depsList, dependency) {
					depsList = append(depsList, dependency)
				}
			}
		}
	}
	return
}
```
no use for technologiesList, can be ommited